scripts/prepdocs.py

```diff
@@ -379,17 +402,4 @@ def refresh_openai_token():
             create_search_index()
 
         print("Processing files...")
-        for filename in glob.glob(args.files):
-            if args.verbose: print(f"Processing '{filename}'")
-            if args.remove:
-                remove_blobs(filename)
-                remove_from_index(filename)
-            elif args.removeall:
```

Comment:
Did we lose removeall entirely? I expected it to move outside the function.

Reply:
Its in the __main__ body, it was already there before, thats why you dont see it in the Review.

https://github.com/Azure-Samples/azure-search-openai-demo/blob/36c14b6248b4a5c82e80c6682535990651b25bc6/scripts/prepdocs.py#L397-L405